src/libraries/System.Text.RegularExpressions/src/System.Text.RegularExpressions.csproj

```diff
@@ -7,6 +7,8 @@
   <ItemGroup>
     <Compile Include="System\Collections\HashtableExtensions.cs" />
     <Compile Include="System\Collections\Generic\ValueListBuilder.Pop.cs" />
+    <Compile Include="System\Text\RegularExpressions\Symbolic\DerivativeEffect.cs" />
+    <Compile Include="System\Text\RegularExpressions\Symbolic\SparseIntMap.cs" />
```

Comment:
NIT: can we move these two files to the bellow NonBacktracking section? (Don't reset CI just for this, but do add it in case you plan to add any other commits.)